app/presenters/tree_node/menu/item.rb

```diff
@@ -0,0 +1,39 @@
+module TreeNode
+  module Menu
+    class Item < TreeNode::Menu::Node
+      set_attribute(:key)     { "#{@options[:node_id_prefix]}__#{@object.feature}" }
+      set_attribute(:title)   { _(details[:name]) }
+      set_attribute(:tooltip) { _(details[:description]) || _(details[:name]) }
+
+      def select_state
+        self_selected? || select_by_children_selected
+      end
+
+      def children
```

Comment:
@hayesr Maybe `:three_checks => true` in `set_locals_for_render` can handle node selection based on selection of its kids :)